### ISSUE TITLE
mongodbatlas_alert_configuration notification microsoft_teams_webhook_url is always updated

### ISSUE THREAD

**Issue by smokedlinq** [mongodb/terraform-provider-mongodbatlas#1275]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
Terraform v1.5.1
on windows_amd64
```

### Terraform Configuration File 

```hcl
terraform {
  required_version = ">=1.5"

  required_providers {
    mongodbatlas = {
      source  = "mongodb/mongodbatlas"
      version = "1.10.0"
    }
  }
}

variable "project_id" {
  type = string
}

variable "microsoft_teams_webhook_url" {
  type      = string
  sensitive = true
}

resource "mongodbatlas_alert_configuration" "high_memory" {
  project_id = var.project_id
  enabled    = true
  event_type = "OUTSIDE_METRIC_THRESHOLD"

  metric_threshold_config {
    metric_name = "SYSTEM_MEMORY_PERCENT_USED"
    operator    = "GREATER_THAN"
    threshold   = 85
    mode        = "AVERAGE"
    units       = "RAW"
  }

  notification {
    type_name                   = "MICROSOFT_TEAMS"
    interval_min                = "60"
    delay_min                   = "15"
    sms_enabled                 = false
    email_enabled               = false
    microsoft_teams_webhook_url = var.microsoft_teams_webhook_url
  }
}
```

### Steps to Reproduce

1. `terraform init`
2. `terraform apply`
3. `terraform apply`

### Expected Behavior

No change should be detected.

### Actual Behavior

```
# mongodbatlas_alert_configuration.high_memory will be updated in-place
  ~ resource "mongodbatlas_alert_configuration" "high_memory" {
        id                     = "ommitted"
        # (6 unchanged attributes hidden)

      ~ notification {
          ~ microsoft_teams_webhook_url = (sensitive value)
            # (6 unchanged attributes hidden)
        }

        # (2 unchanged blocks hidden)
    }
```

**Comment by martinstibbe** [comment]
@smokedlinq Able to replicate issue will fix in [INTMDB-896](https://jira.mongodb.org/browse/INTMDB-896)

**Comment by andreaangiolillo** [comment]
Hello 👋 
We released [v1.10.2](https://registry.terraform.io/providers/mongodb/mongodbatlas/1.10.2) with the fix to this bug.

Thanks
